src/landsat_importer/landsat_metadata.py

```diff
@@ -112,4 +102,17 @@ def __contains__(self, key):
         if self[key]:
             return True
 
+    def get_extent(self, is_lat):
+        # order "UL", "UR", "LL", "LR"
+        root = "LANDSAT_METADATA_FILE/PROJECTION_ATTRIBUTES"
+        lat_or_lon = "LAT" if is_lat else "LON"
```

Comment:
if we're refactoring I'd be tempted to change the function to:
```python
def get_extent(self, latlon):
    assert latlon.upper() in ['LAT', 'LON']
    ul = self[f"{root}/CORNER_UL_{latlon.upper()}_PRODUCT"]
```
so you can write
lats = self.landsat_metadata.get_extent('lat')